src/plugin/updateLocale/index.js

```diff
@@ -5,7 +5,16 @@ export default (option, Dayjs, dayjs) => {
     if (!localeConfig) return
     const customConfigKeys = customConfig ? Object.keys(customConfig) : []
     customConfigKeys.forEach((c) => {
-      localeConfig[c] = customConfig[c]
+      const originConfig = localeConfig[c]
+      const customOriginConfig = customConfig[c]
+      if (originConfig !== null && typeof originConfig === 'object' && typeof customOriginConfig === 'object') {
+        localeConfig[c] = {
+          ...originConfig,
```

Comment:
Thanks for the PR.

Day.js is a lib targeting es5, please do not use es6 syntax like `...` (Destructuring assignment) or `Object.assign` in the code. This will lead significant bundle size increase.

Please consider using some simple code like `for` loop or `Object.keys` to iterate the object.